pkg/netpol/connlist/connlist.go

```diff
@@ -375,6 +381,32 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine) ([]Peer2Pe
 		return nil, err
 	}
 
+	// compute connections between peers based on pe policies rules
+	peersAllowedConns, err := ca.getConnectionsBetweenPeers(pe, peerList)
+	if err != nil {
+		ca.errors = append(ca.errors, newResourceEvaluationError(err))
+		return nil, err
+	}
+	res = append(res, peersAllowedConns...)
+
+	if ia.IsEmpty() {
+		return res, nil
+	}
```

Comment:
there could be services without any configured Ingress/Route resources, in which case we also do not need to add the ingress analysis..